source/_integrations/mcp.markdown

```
@@ -74,6 +79,12 @@ agent to use the tools.
 perform actions through Home Assistant. The tools used by the configured LLM API
 are exposed.
 
+### Authorization
+
+The Model Context Protocol supports OAuth, and allows you to give Home Assistant 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)